disttask/importinto/subtask_executor.go

```diff
@@ -43,13 +44,20 @@ import (
 // TestSyncChan is used to test.
 var TestSyncChan = make(chan struct{})
 
-// ImportMinimalTaskExecutor is a minimal task executor for IMPORT INTO.
-type ImportMinimalTaskExecutor struct {
+// importMinimalTaskExecutor is a minimal task executor for IMPORT INTO.
+type importMinimalTaskExecutor struct {
 	mTtask *importStepMinimalTask
 }
 
-// Run implements the MiniTaskExecutor.Run interface.
-func (e *ImportMinimalTaskExecutor) Run(ctx context.Context) error {
+var newImportMinimalTaskExecutor = newImportMinimalTaskExecutor0
+
+func newImportMinimalTaskExecutor0(t *importStepMinimalTask) execute.MiniTaskExecutor {
```

Comment:
What is the purpose to use `var` instead of function?

Reply:
for ut, see encode_and_sort_operator_test.go